src/resources/nfts/index.ts

```diff
@@ -86,6 +86,6 @@ export function useLegacyNFTs({ address }: { address: string }) {
   return {
     data: nfts,
     error,
-    isFetching,
+    isInitialLoading: !nfts.length && isFetching,
```

Comment:
`isFetching` flickers back and forth between true and false as each page is fetched. i replaced it with `isInitialLoading`, which returns true if we don't have any NFTs yet but are trying to fetch some